.github/README.md

```diff
@@ -44,16 +46,14 @@ $ TFENV_ARCH=amd64 tfenv install 1.3.3
 $ tfenv use 1.3.3
 ```
 
-6. Terraform will ask you if you want to perform these actions: enter yes.
+7. Terraform will ask you if you want to perform these actions: enter yes.
 
-7. Congrats! You created a new custom rule that you can see in your dashboard.
+8. Congrats! You created a new custom rule that you can see in your dashboard.
 
 ## Make a release
 
-//TODO: update the process after making the first release
-
-1. Create a tag with the version number, starting with `v`.
+1. On the main branch, create a tag with the version number, starting with `v`.
```

Comment:
Is the 'v' a requirement for terraform ?
In our internal guidelines we agreed to remove it.

Reply:
+1 👍 💯